sdk/communication/communication-identity/tests.yml

```diff
@@ -5,7 +5,10 @@ stages:
     parameters:
       PackageName: "@azure/communication-identity"
       ServiceDirectory: communication
-      EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+      CloudConfig:
+        Public:
+          SubscriptionConfigurations:
+            - $(sub-config-azure-cloud-test-resources)
+            - $(sub-config-communication-services-cloud-test-resources-test)
```

Comment:
this is renamed to sub-config-communication-services-cloud-test-resources-common